test/router.spec.js

```diff
@@ -539,10 +574,11 @@ describe('createCentralRouter()', () => {
     describe('Data Collector', () => {
       beforeEach(() => {
         mockLogin({ role: 'none' });
-        testData.extendedProjects.createPast(1, { role: 'formfill', forms: 1 });
+        testData.extendedProjects.createPast(1, {
+          role: 'formfill',
+          datasets: 1
+        });
         testData.extendedForms.createPast(1);
-        testData.extendedFormVersions.createPast(1, { draft: true });
-        testData.standardFormAttachments.createPast(1);
```

Comment:
In these tests, the only data that is returned is the data for /projects/1. That means that we don't need to create this form-level data.